src/Symfony/Component/Form/Tests/FormRegistryTest.php

```diff
@@ -173,18 +173,6 @@ public function testGetTypeConnectsParentIfGetParentReturnsInstance()
     }
 
     /**
-     * @expectedException \Symfony\Component\Form\Exception\UnexpectedTypeException
-     */
-    public function testGetTypeThrowsExceptionIfParentNotFound()
```

Comment:
this tests doesn't do what it says. and it would fail in 3.0 even if changed because there the parent type would be created on the fly by with the FQCN.